Reqres.xcodeproj/project.pbxproj

```diff
@@ -127,6 +131,8 @@
 			isa = PBXGroup;
 			children = (
 				69A899D31FF4245D003D0A71 /* Reqres.swift */,
+				7C460A8C214FE8FD00767D3F /* ReqResSwizzle.h */,
```

Comment:
We'd love _Reqres_ to be Swift-only as long as possible, so it would be nice if it would be possible to implement swizzling in Swift.

Reply:
The swizzling method described in 
https://blog.newrelic.com/engineering/right-way-to-swizzle/ isn't available in Swift (as it requires C).

It may work with the swizzling available in Swift, but it may be less stable or break with changes in iOS internals.

Reply:
Okay, but you know we're a bit cautious when referring to 4 years old source